components/PageNavigation.vue

```diff
@@ -227,4 +251,36 @@
       }
     }
   }
+
+  ::v-deep .dropdown {
+    &.nav-link {
+      padding: 1px 0 1px 0;
+    }
+    .label {
+      color: $mediumgrey;
+      font-size: $font-size-small;
+      font-weight: 600;
+      text-decoration: none;
+      text-transform: uppercase;
+    }
+    &-divider {
+      margin: 0;
+    }
+    &-menu {
+      margin-top: 0.65rem;
+      // padding: 0.4rem 0 0.4rem 0;
+      border-radius: 4px;
+      box-shadow: 2px 2px 4px 0 rgba(0, 0, 0, 0.2);
+      border: solid 1px #dcdcde;
+      li a {
+        padding: 0.4rem 1.2rem 0.4rem 1.2rem;
+        &:hover {
+          background-color: #dcdcde;
+        }
+        .label {
+          height: 2.4rem;
```

Comment:
this is an unusual way to style this, maybe it will be fine but there could be cross-browser bugs. Using padding is bulletproof, so remove this style and on `li a` change padding to `0.85rem 1rem` this will make it 48px. Currently height 2.4rem is making it 53px, the design is 48px.